### ISSUE TITLE
`AnimationTrackKeyEdit` time value undo/redo history is not recorded correctly when using grabber/slider

### ISSUE THREAD

**Issue by TokageItLab** [godotengine/godot#69529]
### Godot version

3.x, 4.0.dev

### System information

any

### Issue description

AnimationTrackKeyEdit sets a property to a dummy object and then sets the value again to the actual Animation resource, but the `track_remove_key_at_time()` in it must specify a time.

However, when the time value is set by SpinSlider's grabber/slider, it is updated every frame, and UndoRedo does not correctly record the needed time, and Undo does not correctly remove keys that need to be erased.

https://user-images.githubusercontent.com/61938263/205433351-a140fe5a-e33d-4d17-a1b2-8a2dd8d74f97.mp4

Even if the time is set correctly or the keys are deleted by idx, when the keys overlap during the updating of the frame values by grabbing/sliding in SpinSlider, the overlapped keys are deleted implicitly and cannot be recorded in the history.

In other words, in order to properly implement time editing history in the inspector, it is necessary to commit the action only when the grabber/slider is released, similar to AnimationTimelineEdit, instead of setting the value every frame.

To accomplish this, AnimationTrackKeyEdit would need to be fundamentally redesigned.

---

However, this bug can be avoided by disabling grabber altogether. It is definitely a naive workaround. But actually, most key time editing is done in AnimationTimelineEdit, and the inspector edit is there for direct input of values which should be the use case for 99% of the cases. So, I think it is fine...

### Steps to reproduce

Edit time value in AnimationTrackKeyEdit with grabber/slider

### Minimal reproduction project

N/A
